### PR TITLE
fix:(Disabled the automatic verification; If you wana be verified keep calm)

### PR DESCRIPTION
**Pull request by Juliaria08**
When the user verificate automaticate revoke the verificated role

### CODE REVIEW

**Comment by Juliaria08** [comment]
Yes thats the way to disable it when you disable if any stupid tries to veryficate when they are alredy verificated this check unverificates itself and sends a fancy message.

**Comment by Juliaria08** [comment]
@danirod

File: src/hooks/verify.ts [hunk 1]
```diff
@@ -45,7 +49,7 @@ export default class VerifyService implements Hook {
       throw new ReferenceError(`Verification role not found in guild ${message.guild.name}!`);
     }
     message.member
-      .addRole(role)
+      .delRole(role)
```
Comment (danirod) [inline src/hooks/verify.ts:52]: There is no point on using `delRole` because `message.member` will lack the role at that point anyway.

**Comment by Juliaria08** [comment]
Ok

**Pull request closed without merging**